python/ray/package.py

```diff
@@ -0,0 +1,90 @@
+import hashlib
```

Comment:
Please move this file to ray._private

Reply:
Sure, but I still need to expose some API here for users to create their own package.

Reply:
Won't the only API be the specification of `runtime_env` in ray.init() for now?

Reply:
+1 for only supporting specifying it in `ray.init` for now

Reply:
I'm good with this.